<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="16" hostname="Aditya" failures="7" tests="12" name="NopCommerce Tests" time="320.189" errors="0" timestamp="2025-09-08T17:00:55 IST">
  <testcase classname="com.capstone.tests.LoginTest" name="loginTestBlankCredentials" time="3.859"/>
  <testcase classname="com.capstone.tests.LoginTest" name="loginTestInvalidEmail" time="25.35"/>
  <testcase classname="com.capstone.tests.LoginTest" name="loginTestInvalidPassword" time="25.528"/>
  <testcase classname="com.capstone.tests.LoginTest" name="loginTestValidCredentials" time="25.341">
    <failure type="java.lang.AssertionError" message="Login failed: Login was unsuccessful. Please correct the errors and try again.
No customer account found expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: Login failed: Login was unsuccessful. Please correct the errors and try again.
No customer account found expected [false] but found [true]
at com.capstone.tests.LoginTest.loginTestValidCredentials(LoginTest.java:29)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginTestValidCredentials -->
  <testcase classname="com.capstone.tests.RegisterTest" name="registerNewUser" time="21.74"/>
  <testcase classname="com.capstone.tests.AddressTest" name="addNewAddress" time="15.513">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.AddressTest.addNewAddress(AddressTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addNewAddress -->
  <testcase classname="com.capstone.tests.CartTest" name="cartTest" time="15.531">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.CartTest.cartTest(CartTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- cartTest -->
  <testcase classname="com.capstone.tests.ChangePasswordTest" name="PasswordTest" time="16.021">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.ChangePasswordTest.PasswordTest(ChangePasswordTest.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- PasswordTest -->
  <testcase classname="com.capstone.tests.CheckoutTest" name="cartTest" time="15.909">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.CheckoutTest.cartTest(CheckoutTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- cartTest -->
  <testcase classname="com.capstone.tests.E2ELoginTest" name="testEndToEndEcommerceFlow" time="15.462">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.E2ELoginTest.testEndToEndEcommerceFlow(E2ELoginTest.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testEndToEndEcommerceFlow -->
  <testcase classname="com.capstone.tests.E2ERegisterTest" name="testEndToEndEcommerceFlow" time="95.891"/>
  <testcase classname="com.capstone.tests.ProductSearchTest" name="loginTest" time="15.345">
    <failure type="java.lang.AssertionError" message="Login failed! expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login failed! expected [true] but found [false]
at com.capstone.tests.ProductSearchTest.loginTest(ProductSearchTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@BeforeSuite setupReport" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeSuite setupReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
  <testcase classname="com.capstone.framework.core.BaseTest" name="@AfterSuite tearDownReport" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterSuite tearDownReport -->
</testsuite> <!-- NopCommerce Tests -->
